pkg/preparer/orchestrate.go

```diff
@@ -212,6 +208,14 @@ func (p *Preparer) verifySignature(manifest pods.Manifest, logger logging.Logger
 func (p *Preparer) installAndLaunchPod(newManifest *pods.Manifest, pod Pod, logger logging.Logger) bool {
 	// do not remove the logger argument, it's not the same as p.Logger
 
+	// HACK ZONE. When we have better authz, rewrite.
+	if (newManifest.RunAsUser() == "root") && (newManifest.ID() != POD_ID) {
```

Comment:
The existing test is `p.podIdForbidden(newManifest.ID())`. But that same test needs to apply to `RunAs`.